openvinotoolkit/datumaro
# Suggested fix for issue #560

### Summary

Resolves issue (bug) #560
With this fix, the following issue in CVAT will also be resolved and does not need further changes: https://github.com/openvinotoolkit/cvat/issues/3940

### How to test

This was tested manually through the CVAT tool:

1. Create a new project and upload a video
2. Open the task
3. Click Actions -> upload annotations -> MOT 1.1
4. Select zip folder with MOT annotation format (including seqinfo.ini file) and click 'upload'

With this PR, importing should succeed

### Checklist

- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
